internal/config/config.go

```diff
@@ -121,6 +121,19 @@ type AppConfig struct {
 		// Disabled by default.
 		Domain string `yaml:"domain"`
 	} `yaml:"dns"`
+	// Configures settings in the configuration file distributed to clients, either by download, or QR-code.
+	ClientConfig struct {
+		// DNS servers to be provided with the client configuration file.
+		// These are written into the configuration file as is.
+		// If left empty the server decides about the address; usually the wg-access-server address.
+		// If not empty, these replace the wg-access-servers DNS addresses.
+		// Empty by default.
+		DnsServers []string `yaml:"dnsServers"`
```

Comment:
In go initialisms [should be uppercase](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

Hence `DNSServers`

Reply:
:+1: